hedera-node/hedera-app/src/main/java/com/hedera/node/app/records/BlockRecordService.java

```diff
@@ -46,6 +47,8 @@ public final class BlockRecordService implements Service {
     /** The original hash, only used at genesis */
     private static final Bytes GENESIS_HASH = Bytes.wrap(new byte[48]);
 
+    public static final Timestamp EPOCH = new Timestamp(0, 0);
```

Comment:
We could probably add some description here for this saying we use it only on GENESIS ?

Reply:
Ah oops missed this. 😞 Adding to a following PR now.